Common/tests/Pollster/PollsterTest.cs

```diff
@@ -480,8 +483,13 @@ await testServiceProvider.Pollster.Init(CancellationToken.None)
                              .ConfigureAwait(false);
 
     var stop = testServiceProvider.StopApplicationAfter(TimeSpan.FromSeconds(1));
+    var loop = testServiceProvider.Pollster.MainLoop();
 
-    Assert.DoesNotThrowAsync(() => testServiceProvider.Pollster.MainLoop());
+    // checking health should not interrupt task execution
+    var res = await testServiceProvider.Pollster.Check(HealthCheckTag.Liveness)
```

Comment:
You should most likely not affect the result of the check to a new variable that is not used anyway.